Source/WebCore/html/HTMLElement.h

```diff
@@ -88,6 +88,7 @@ class HTMLElement : public StyledElement {
     WEBCORE_EXPORT const AtomString& dir() const;
     WEBCORE_EXPORT void setDir(const AtomString&);
 
+    TextDirection computeDirectionality() const;
     bool hasDirectionAuto() const;
     TextDirection directionalityIfhasDirAutoAttribute(bool& isAuto) const;
```

Comment:
Pre-existing issue, and unrelated to your patch, but the "has" in this function name should probably be "Has".